community/bat/build

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+cargo build --release
+
+install -Dm755 target/release/bat "$1/usr/bin/bat"
+find . -name bat.1 -exec install -Dm644 {} \
```

Comment:
Do we not know where the file is?

Reply:
The man-page is generated at a path that looks like this:
`./target/release/build/bat-484c9d3dc1d6eff0/out/assets/manual/bat.1`
If i compile the git version then the path is like this:
`./target/release/build/bat-ef405f4526930eea/out/assets/manual/bat.1`
Hence, `find` is used as the man-page path will probably change over new releases

Reply:
```sh
install -Dm644 target/release/build/bat-*/out/assets/manual/bat.1 "$1/usr/share/man/man1"
```

Does this work?

Reply:
Can we not install everything using `cargo`?

Reply:
Cargo doesn't seem to install the man-page, only the executable is installed.
I've added a commit to use the method you suggested for installing the man-page.

Reply:
Seems like cargo doesn't yet support installing man-pages either, so it'll have to be done manually. See https://github.com/rust-lang/cargo/issues/2729